src/environments/environment.test.ts

```diff
@@ -2,45 +2,65 @@
 import { BrowseByType } from '../app/browse-by/browse-by-switcher/browse-by-decorator';
 import { RestRequestMethod } from '../app/core/data/rest-request-method';
 import { NotificationAnimationsType } from '../app/shared/notifications/models/notification-animations-type';
-import { GlobalConfig } from '../config/global-config.interface';
+import { AppConfig } from '../config/app-config.interface';
 
-export const environment: Partial<GlobalConfig> = {
-  rest: {
-    ssl: true,
-    host: 'rest.com',
-    port: 443,
-    // NOTE: Space is capitalized because 'namespace' is a reserved string in TypeScript
-    nameSpace: '/api',
-    baseUrl: 'https://rest.api/'
+export const environment: AppConfig = {
```

Comment:
While I'm not against keeping the Test environment configuration in Typescript, it might be better to see if we could also move these to YAML...as that way the Test environment is more similar to normal dev/prod environments.  Would there be a way to move this to YAML (either in this PR or in follow-up work)?

Reply:
Is this necessary for unit tests?

Reply:
I am for a consistent way to express the config. I am not sure how the unit tests differ between dev/prod environments. The integration tests use the prod configs.

Reply:
The unit tests do not run building the config. The simply provide it in the spec. For simplicity it makes sense to keep the unit test config in environment.test.ts which is only used during unit tests. Alternatively, it should not use any file and provide the used config properties only in each spec. Isolating even the configuration.

Reply:
@wwelling : Ok, I misunderstood...for some reason I had assumed `environment.test.ts` was used by *both* unit and integration tests.  If it's *just* for unit tests, then this is fine as-is.  I just wanted to be sure that we had tests which proved this new YAML runtime config build process continues to work properly....so if that's the e2e tests, that's good enough for me.